### PR TITLE
[File System Access article] Add serialization instructions

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add serialization instructions.

### CODE REVIEW

**Comment by tomayac** [comment]
(Triggered via a [StackOverflow question](https://stackoverflow.com/questions/65928613/file-system-access-api-is-it-possible-to-store-the-filehandle-of-a-saved-or-loa/65938910#65938910) and https://github.com/GoogleChromeLabs/browser-fs-access/issues/30.)

**Comment by kaycebasques** [comment]
Hello! This is an automated review. It updates automatically when code or GitHub comments in this pull request are created or updated.

## Staging URLs

For your convenience, here are direct links (on our staging site) to the content you created or updated:

* https://deploy-preview-4550--web-dev-staging.netlify.app/file-system-access/
## Requested changes

If there are any common problems with the content files you created or modified, they will be listed here.

### `src/site/content/en/blog/file-system-access/index.md`

* This file passed all of our automated Markdown audits.

File: src/site/content/en/blog/file-system-access/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ description:
   user grants a web app access, this API allows them to read or save changes directly to files and
   folders on the user's device.
 date: 2019-08-20
-updated: 2020-11-10
+updated: 2021-01-28
```
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:15]:
```suggestion
updated: 2021-02-02
```

**Comment by tomayac** [comment]
@kaycebasques @jpmedley This should be good to go. Thank you!

File: src/site/content/en/blog/file-system-access/index.md [hunk 2]
```diff
@@ -233,6 +233,37 @@ working on. This makes it possible to keep a list of recently opened or edited f
 re-open the last file when the app is opened, etc. In the text editor, I store a list of the five
 most recent files the user has opened, making it easy to access those files again.
 
+The code snippet below shows storing and retrieving a file handle at an example.
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:236]:
```suggestion
The code example below shows storing and retrieving a file handle.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by tomayac** [comment]
@jpmedley Comment addressed. Please approve and merge at your convenience.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**